README.md

```diff
@@ -8,12 +10,14 @@ This library enhances [`styled-components`](https://www.styled-components.com/)
 
 This package is available on npm as `styled-components-modifiers`, and you can find it [here](https://www.npmjs.com/package/styled-components-modifiers).
 
-To install the latest, stable version with `npm` run:
+To install the latest stable version with `yarn`:
+
 ```sh
 $ npm install styled-components-modifiers --save
 ```
```

Comment:
you should switch the scripts too, @tinynumbers

Reply:
Yeah, already caught that. 👍